delivery_carrier_state/README.rst

```diff
@@ -0,0 +1,102 @@
+======================
+Delivery carrier state
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
Just use readme parts that generate the readme automatically :sweat_smile: 
Put the fragments you need in the readme folder and OCA bot will do the rest (or you can generate it in advance with https://github.com/OCA/maintainer-tools#readme-generator wich prevents errors). Estructure template:
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
@chienandalu Thanks for the info, I do it this way